category.go

```diff
@@ -1,49 +1,56 @@
 package cli
 
-// CommandCategories is a slice of *CommandCategory.
+// CommandCategories wraps a slice of *CommandCategory.
 type CommandCategories struct {
```

Comment:
One downside is that there isn't a good way to remove a category (admittedly this is probably an edge case for normal usage).